pkg/types/options.go

```diff
@@ -90,3 +94,17 @@ func (options *Options) ParseCustomHeaders() map[string]string {
 	}
 	return customHeaders
 }
+
+func (options *Options) ParseHeadlessOptionalArguments() map[string]string {
+	optionalArguments := make(map[string]string)
+	for _, v := range options.HeadlessOptionalArguments {
+		if argParts := strings.SplitN(v, "=", 2); len(argParts) >= 2 {
+			key := strings.Trim(argParts[0], " ")
```

Comment:
I think here we can use `strings.TrimSpace`

Reply:
✔️